source/renderer/app/components/wallet/send-form/AssetInput.js

```diff
@@ -101,119 +85,100 @@ export default class AssetInput extends Component<Props> {
-              amount={quantity}
-              metadata={metadata}
-              decimals={decimals}
-            />
-          </div>
-        )}
-        <NumericInput
-          {...assetField.bind()}
-          ref={(field) => addFocusableField(field)}
-          placeholder={
-            decimals
-              ? `0${currentNumberFormat.decimalSeparator}${'0'.repeat(
-                  decimals
-                )}`
-              : '0'
-          }
-          className={styles.assetItem}
-          label={
-            <>
-              {`${intl.formatMessage(messages.assetLabel)} #${index + 1}`}
-              {removeAssetButtonVisible[uniqueId] && (
-                <span
-                  className={classNames([styles.removeAssetButton, 'flat'])}
-                  onClick={() => removeAssetRow(uniqueId)}
-                >
-                  {intl.formatMessage(messages.removeLabel)}
-                </span>
-              )}
-            </>
-          }
-          bigNumberFormat={decimals ? currentNumberFormat : null}
-          decimalPlaces={decimals}
-          numberLocaleOptions={{
-            minimumFractionDigits: decimals,
-          }}
-          onChange={(value) => {
-            assetField.onChange(value);
-          }}
-          currency={ticker}
-          value={assetField.value}
-          error={assetField.error}
-          skin={AmountInputSkin}
-          style={inputFieldStyle}
-          onKeyPress={(evt: SyntheticKeyboardEvent<EventTarget>) => {
-            if (decimals === 0) {
-              const { charCode } = evt;
-              if (charCode === 190 || charCode === 110 || charCode === 46) {
-                evt.persist();
-                evt.preventDefault();
-                evt.stopPropagation();
-              }
-            }
-            handleSubmitOnEnter(evt);
-          }}
-          allowSigns={false}
-          autoFocus={autoFocus}
-        />
-        <div className={styles.rightContent} ref={this.rightContentRef}>
-          {this.hasAssetValue(assetField) && (
-            <div className={styles.clearAssetContainer}>
-              <PopOver
-                content={intl.formatMessage(messages.clearLabel)}
-                placement="top"
-              >
-                <button
-                  onClick={() => clearAssetFieldValue(assetField)}
-                  className={styles.clearAssetButton}
-                >
-                  <SVGInline
-                    svg={closeIcon}
-                    className={styles.clearReceiverIcon}
-                  />
-                </button>
-              </PopOver>
-              <div className={styles.separator} />
+      <div key={`receiver_asset_${uniqueId}`} className={styles.component}>
+        <div className={styles.inputBlock}>
+          {quantity.isPositive() && (
+            <div className={styles.amountTokenTotal}>
+              {intl.formatMessage(messages.ofLabel)}
+              {` `}
```

Comment:
rather than using this, why not use the proceeding component, with a css :before { content: " " }?

Reply:
Do you mean like bellow?

```
.someclass {
   &:before { content: " " }
}

<span className="someclass">
   <DiscreetTokenWalletAmount
      amount={quantity}
      metadata={metadata}
      decimals={decimals}
    />
</span>
```

Reply:
Yes 👍

Reply:
Ok, I'll update.